libs/ledger-live-common/package.json

```diff
@@ -278,6 +277,7 @@
     "jest-environment-jsdom": "28",
     "jest-file-snapshot": "^0.5.0",
     "long": "^4.0.0",
+    "mkdirp": "*",
```

Comment:
I don't know what this package is for, but it doesn't seemed used anywhere?

Reply:
Looks like it is used in the bot `libs\ledger-live-common\src\bot\portfolio\process-main.ts`

Reply:
:open_mouth: didn't see it when I searched last time, weird